terraform/vsphere_active_directory/files/install_active_directory.ps1

```diff
@@ -1,12 +1,21 @@
+# install_active_directory turns the VM into an active directory domain controller and creates the relevant forrests.
+# Once installed, the script will intitiate a reboot.
+
 $domainName = "${domain_name}"
 $domainNetbiosName = "${netbios_name}"
 $safeModeAdminstratorPassword = ConvertTo-SecureString "${password}" -AsPlainText -Force
-
-Write-Output "Installing Windows Feature for AD-Domain-Services..."
-
-Install-WindowsFeature -Name AD-Domain-Services -IncludeManagementTools
 Import-Module ADDSDeployment
 
+if ((Get-WindowsFeature -Name DHCP).InstallState -eq "Installed") {
+    try {
```

Comment:
I don't think try/catch will have any effect without setting the `ErrorActionPreference` to something non-default.